src/components/classes.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
`classes`,  `reports`, `students` are not really components, is there a better place to put it?